src/lighteval/metrics/metrics_sample.py

```diff
@@ -625,22 +625,25 @@ class JudgeLLM:
 
     def __init__(self, judge_model_name: str, template_path: str, multi_turn: bool = False):
         if judge_model_name not in self.available_models:
-            raise ValueError(f"{judge_model_name} not in available models for llm as a judge metric")
+            judge_type = "openai"
```

Comment:
this wouldn't work, if we pass in `gpt-12` for example, it will set the judge type to `openai` and continue, only to fail later because `gpt-12` does not exist in the `openai` api